ksql-engine/src/main/java/io/confluent/ksql/physical/PhysicalPlanBuilder.java

```diff
@@ -212,9 +212,9 @@ private QueryMetadata buildPlanForStructuredOutputNode(
     }
     final StructuredDataSource sinkDataSource;
     if (schemaKStream instanceof SchemaKTable) {
```

Comment:
wish this was a method, too ...

Reply:
Agreed, but trying to minimise changes... its a big PR already